### PR TITLE
Layout array renaming

### PR DESCRIPTION
**Pull request by GernotMaier**
This is a follow up to #428 to change the term `LayoutArray` to `ArrayLayout` through the code (also `layout_array` to `array_layout`). 

There is zero functional change, this is search and replace only. So I think that no code review is required (as long as all tests pass).

### CODE REVIEW

File: rnda-docker.sh
Comment (orelgueta) [inline rnda-docker.sh:1]: Should this file be included in this PR (or in general)?

File: rnda.sh
Comment (orelgueta) [inline rnda.sh:1]: Should this file be included in this PR (or in general)?

**Comment by orelgueta** [comment]
Also, there are conflicts apparently, probably cause it took me so long to review this PR...

File: rnda-docker.sh
Comment (GernotMaier) [reply rnda-docker.sh:1]: No!!

File: rnda.sh
Comment (GernotMaier) [reply rnda.sh:1]: No!!

**Comment by GernotMaier** [comment]
Fixed both things - remove the accidentally committed shell scripts and merged main into this branch.

**Pull request merged**